opentelemetry-api/src/opentelemetry/configuration/__init__.py

```diff
@@ -167,3 +179,20 @@ def _reset(cls) -> None:
         if cls._instance:
             cls._instance._config_map.clear()  # pylint: disable=protected-access
             cls._instance = None
+
+    def _traced_request_attrs(self, instrumentation: str) -> List[str]:
+        """Returns list of traced request attributes for instrumentation."""
+        key = "{}_TRACED_REQUEST_ATTRS".format(instrumentation.upper())
+        value = self._config_map.get(key, "")
+
+        request_attrs = (
+            [attr.strip() for attr in str.split(value, ",")] if value else []  # type: ignore
+        )
+        return request_attrs
+
+    def _excluded_urls(self, instrumentation: str) -> ExcludeList:
+        key = "{}_EXCLUDED_URLS".format(instrumentation.upper())
+        value = self._config_map.get(key, "")
```

Comment:
I'm not sure if commas are allowed in URLs, but they can appear meaningfully in a regex like `\d{5,10}`. This seems more like an issue with passing comma separated regexes in envars than this PR though.

Reply:
Comma is reserved character and is not typically used in URLs. It is normally used in query component for name value pairs.